tests/test_issue_2458_nested_ssh_term_respects_portable_term.py

```
@@ -0,0 +1,428 @@
+    root: Path,
+    zsh_path: str,
+    env: dict[str, str],
+    command_string: str,
+    login_shell: bool,
+) -> tuple[bool, str]:
+    argv = [zsh_path, "-d"]
+    if login_shell:
+        argv.append("-l")
+    argv.extend(["-i", "-c", command_string])
+
+    result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)